exercises/diamond/diamond.py

```diff
@@ -1,2 +1,2 @@
-def make_diamond():
+def make_diamond(string):
```

Comment:
`string` is too general and judging by https://github.com/exercism/python/blob/master/exercises/diamond/diamond_test.py only one letter is expected

Reply:
My initial thought was to make the placeholder the word 'letter' or 'character', but decided on string since the single letter is still expected in string format I went with 'string' to illustrate the type. I can certainly update it to something more specific such as 'letter'.